.github/ISSUE_TEMPLATE/missing_documentation.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Missing or incorrect documentation
+about:
+title: ''
+labels: 'documentation'
+assignees: ''
+
+---
+
+## Problem
+
+<!-- describe your problem -->
+
+## Checklist
+
+<!-- make sure this issue is not redundant or obsolete -->
+
+- [ ] checked [latest Nix manual]\ ([source])
```

Comment:
```suggestion
- [ ] checked [latest Nix manual][latest Nix manual] ([source])
```

I assume the `\` was here to prevent it from being interpreted as an inline link, but it shows up in the rendered markdown. Expliciting the reference should alleviate that

Reply:
```suggestion
- [ ] checked [latest Nix manual] \([source])
```

This also works and is less noise.

Reply:
Oh, great :+1: